app/views/ui-components/v1/navs/_primary_nav.html.slim

```diff
@@ -92,3 +93,17 @@
               li.pl-1 role="presentation" class="#{pundit_class HbxProfile, :view_admin_tabs?}"
                 = link_to(notifier.notice_kinds_path, "aria-expanded" => "true", "aria-controls" => "notices", "role" => "tab" ) do
                   | Notices
+
+
+javascript:
+  document.addEventListener('DOMContentLoaded', function() {
+    window.localStorage.removeItem("jwt");
+    var agencyStaffLink = document.getElementById('agencyStaffLinkToExternal');
```

Comment:
Use `const` over `var`
- explanation: `var` will check outside of the scope of the current function for set values. `const` will scope the variable to the current function. 99% of the time you would never have an issue with a var scope leaking but with `const` you never will.